BaseStation/MainWindow/MapDisplay.xaml.cs

```diff
@@ -39,10 +42,24 @@ public void DisplayMap(string mapSetFile)
                     string line = file.ReadLine();
                     string[] config = line.Split('|');
                     string[] imgDim = config[0].Split('x');
-                    int imgWidth = 1;
-                    int imgHeight = 1;
-                    Int32.TryParse(imgDim[0], out imgWidth);
-                    Int32.TryParse(imgDim[1], out imgHeight);
+                    string[] centerCoords = config[1].Split(',');
+
+                    ImageWidth = 1;
+                    ImageHeight = 1;
+                    double MapCenterLat = 0;
+                    double MapCenterLong = 0;
+                    Zoom = 1;
+
+                    Int32.TryParse(config[2], out Zoom);
```

Comment:
I just realized there's no error handling if this parse operation goes wrong (for whatever reason) Could you add some check that this succeeded? It might look like anding all the results together and failing gracefully instead of just silently ignoring the values